controllers/postComment.js

```diff
@@ -0,0 +1,37 @@
+const { insertComment } = require("../models/insertComment");
+const { fetchArticleById } = require("../models/article_id");
+
+exports.postComment = (req, res, next) => {
+  const { username, body } = req.body;
+  const article_id = req.params.article_id;
+
+  if (isNaN(article_id)) {
+    const err = new Error("Bad request");
+    err.status = 400;
+    throw err;
+  }
+  if (!username || !body) {
+    const err = new Error("Bad request");
+    err.status = 400;
+    throw err;
+  }
+
+  fetchArticleById(article_id)
+    .then((article) => {
+      if (!article) {
+        const err = new Error("Not found");
+        err.status = 404;
+        throw err;
+      }
+    })
+    .then(() => {
+      insertComment({ username, body }, article_id).then((comment) => {
+        res.status(201).send({
+          comment,
```

Comment:
We only need to send back the comment here, rather than the message. We can tell it was successfully added by the status code